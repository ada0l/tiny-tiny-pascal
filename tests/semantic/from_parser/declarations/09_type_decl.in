type i = Integer;
type r = Real; s = String;
begin end.
