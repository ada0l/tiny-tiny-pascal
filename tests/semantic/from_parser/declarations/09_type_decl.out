(2, 10): Not found type